### PR TITLE
Sign up login pages

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# SignUp And Login Pages
## Added 
- signUp page  
- login page 

scrnli_3_27_2022_1-21-01 AM
scrnli_3_27_2022_1-21-43 AM

### CODE REVIEW

**Review by Asuraking1n — APPROVED** [review]
Looks good to me

File: src/pages/SignUp/SignUp.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+              className="input-name sample-input-email"
+              type="text"
+              id="first-name-input"
+              placeholder="Enter your First Name"
+              required
+            />
+          </div>
+          <div className="flex-col">
+            <label className="signup-labels " htmlFor="last-name-input">
+              {" "}
```
Comment (arulk8) [inline src/pages/SignUp/SignUp.jsx:24]: pls remove this.

**Review by arulk8 — COMMENTED** [review]
Good to go

File: src/pages/SignUp/SignUp.jsx (hunk 1 above)
Comment (dusariudaykumar) [reply src/pages/SignUp/SignUp.jsx:24]: 😅 sure

**Pull request merged**